index.html

```diff
@@ -0,0 +1,188 @@
+    <link rel="stylesheet" href="./main.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <h2>Regiss M.</h2>
+            <div><img src="./Images/Icon-Menu.svg" alt="menu"></div>
+        </nav>
+        <div class="header-title">
+            <h1>Hey There. I’m Regiss</h1>
+            <h3>I'am software developer</h3>
+            <p>I can help you build a product. feature or website look through some of my work and experience! if you
+                like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <div class="logo-plat">
+                <ul>
+                    <li><a href="https://github.com/Regiss05" target=”_blank”><img src="./Images/github.svg"
+                                alt="github" /></a></li>
+                    <li><a href="https://www.linkedin.com/in/regiss-mukubiza-1bab841b3/" target=”_blank”><img
+                                src="./Images/linkedin.svg" alt="linkedin" /></a></li>
+                    <li><img src="./Images/Tert.svg" alt="Tert" /></li>
+                    <li><a href="https://twitter.com/regissmukubiza" target=”_blank”><img src="./Images/twitter.svg"
+                                alt="twitter" /></a></li>
+                    <li><a href="https://medium.com/@regiss05" target=”_blank”><img src="./Images/Medium.svg"
+                                alt="Medium" /></a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <div class="container">
+        <div class="container-block">
+            <h1>My Recent Works</h1>
+            <section>
+                <img src="./Images/Placeholder.svg" alt="Placeholder">
+                <h2>Multi-Post Stories</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <div class="box-bot">
+                    <div>CSS</div>
+                    <div>html</div>
+                    <div>bootstrap</div>
+                    <div>Ruby</div>
+                </div>
+                <button type="button">See Project</button>
+            </section>
+            <div>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
```

Comment:
- [ ] Please, use `ul` tag instead of `div` for the list of technologies used and each of them a `li` tag instead of `div`.
- Example:
```
<ul>
   <li></li>
   <li></li>
   <li></li>
</ul>